src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php

```diff
@@ -1283,6 +1283,25 @@ public function testWither()
         $wither = $container->get('wither');
         $this->assertInstanceOf(Foo::class, $wither->foo);
     }
+
+    public function testMultipleDeprecatedAliasesWorking()
```

Comment:
Missing annotations to test the deprecations:

```php
    /**
     * This test checks the trigger of a deprecation note and should not be removed in major releases.
     *
     * @group legacy
     * @expectedDeprecation The "deprecated1" service alias is deprecated. You should stop using it, as it will be removed in the future.
     * @expectedDeprecation The "deprecated2" service alias is deprecated. You should stop using it, as it will be removed in the future.
     */
```